osd_cluster_create/managing-byo-vpc-multiple-az.adoc

```diff
@@ -0,0 +1,29 @@
+[id="managing-byo-vpc-multiple-az"]
+= Managing your own virtual private cloud with multiple availability zones in {product-title}
+:context: byo-vpc-multiple-az
+include::modules/attributes-openshift-dedicated.adoc[]
+
+toc::[]
+
+Configuring a cluster to include your own pre-configured virtual private cloud (VPC) allows for the use of any existing network that relies on an existing VPC.
+
+[id="byo-vpc-multiple-preq"]
+== Prerequisites
+
+* You have a configured VPC.
+* You have a configured AWS account in your AWS organization. The account should have the following:
+.. an IAM user with the name `osdCcsAdmin`.
+.. The `AdministratorAccess` policy attached to this role.
+
+After you have the AWS account ID, you are ready to add a pre-configured VPC.
+
+include::modules/byo-vpc-prep.adoc[leveloffset=+1]
+
+include::modules/byo-vpc-multiple-az.adoc[leveloffset=+1]
+
+include::modules/byo-vpc-multiple-az-ocm-config.adoc[leveloffset=+2]
+
+.Additional resources
+
+* See xref:../osd_quickstart/osd-quickstart.adoc#osd-quickstart[Quick Start] for general cluster creation information.
+* See xref:../osd_cluster_create/managing-byo-vpc-single-az.adoc#managing-byo-vpc-multiple-az[Managing a bring-your-own virtual private cloud with a single availability zone] for instances where only one availability zones are needed.
```

Comment:
```suggestion
* See xref:../osd_cluster_create/managing-byo-vpc-single-az.adoc#managing-byo-vpc-multiple-az[Managing a bring-your-own virtual private cloud with a single availability zone] for instances where only one availability zone is needed.
```
Right?

Reply:
Please double-check that the display text closely matches the target's title